tests/unit/swagger/schemaDetails3.spec.ts

```diff
@@ -126,6 +126,21 @@ describe('Definition generation for OpenAPI 3.0.0', () => {
         });
       });
     });
+    describe('hidden paths', () => {
+      it('should not contain hidden paths', () => {
+        const metadataHiddenMethod = new MetadataGenerator('./tests/fixtures/controllers/hiddenMethodController.ts').Generate();
+        const specHiddenMethod = new SpecGenerator3(metadataHiddenMethod, getDefaultOptions()).GetSpec();
+
+        expect(specHiddenMethod.paths).to.have.keys(['/Controller/normalGetMethod']);
+      });
+
+      it('should not contain paths for hidden controller', () => {
```

Comment:
So basically the same thing you did here but with SpecGenerator2 instead. And just put it anywhere other than this file.